test/e2e/config/vsphere-dev.yaml

```diff
@@ -171,6 +171,17 @@ variables:
   # CAPV feature flags
   EXP_NODE_ANTI_AFFINITY: "true"
   EXP_NODE_LABELING: "true"
+  # Following CAPV variables is used for multivc_test.go. This is the second VSphere and should be set if multivc test is enabled.
+  # VSPHERE2_SERVER: "vcenter2.vmware.com"
+  # VSPHERE2_TLS_THUMBPRINT: "AA:BB:CC:DD:11:22:33:44:EE:FF"
+  # VSPHERE2_RESOURCE_POOL: "ResourcePool"
+  # VSPHERE2_TEMPLATE: "ubuntu-2004-kube-v1.25.6"
+  # Dedicated IP to be used by kube-vip
+  # VSPHERE2_CONTROL_PLANE_ENDPOINT_IP:
+  # Following variables are also required and please use env variables to avoid disclosure of sensitive data
+  # VSPHERE2_USERNAME:
+  # VSPHERE2_PASSWORD:
```

Comment:
Makes sense to keep these in the `dev.yaml` since we cannot run it upstream right now which uses the `vsphere-ci.yaml`.

Reply:
Sure, just uncommented them.